src/main/webapp/app/iris/overview/base-chatbot/iris-base-chatbot.component.ts

```diff
@@ -115,8 +136,12 @@ export class IrisBaseChatbotComponent implements OnInit, OnDestroy, AfterViewIni
     faThumbsUp = faThumbsUp;
     faThumbsDown = faThumbsDown;
     faRedo = faRedo;
+    faPenToSquare = faPenToSquare;
```

Comment:
is it possible to make all of that icons protected readonly ? not neceassary, but follows best practices